### ISSUE TITLE
"simplebar": "^3.0.0-beta.1" bottom scrollbar does not work

### ISSUE THREAD

**Issue by jpfernandezl** [Grsmto/simplebar#179]
When installed via npm, the bottom scrollbar does not work.
"simplebar": "^3.0.0-beta.1"

**Comment by yaymukund** [comment]
I made a bit of progress debugging this, but couldn't get it quite working. At the very least, [`onMouseDown()` needs to check for the click on the horizontal scrollbar here](https://github.com/Grsmto/simplebar/blob/master/packages/simplebar/src/simplebar.js#L497). In addition, I had to change the code to compare against `getBoundingClientRect`'s `top|bottom|left|right` rather than what you're doing now. After that, I found it would scroll but the scrollbar would move past the edge of the screen (so: I guess either a problem with the width or the position or both).

I reset the code by accident, so I can't give any more info but I hope that's helpful :)
